pipeline.yml

```diff
@@ -880,10 +879,9 @@ jobs:
       resource: masterbosh-state
   - task: decrypt-private-key
     file: pipeline-tasks/decrypt.yml
-    config:
-      inputs:
-      - name: pipeline-tasks
-      - name: common-masterbosh
+    input_mapping:
+    - name: pipeline-tasks
```

Comment:
@jmcarp it's because it's throwing the deprecation notice because we're using the `config` and `file` properties together. https://github.com/concourse/atc/blob/master/validate.go#L444-L446